### PR TITLE
fix: Add resolveClient param `clientId`

### LINKED ISSUE #1023: [apollo-composable]  Multiple ApolloClients don't work
**Describe the bug**
`resolveClient `  should has argument `clientId` #906

function resolveClient code:
```
function resolveClient() {
    if (clientId) {
      resolveClientWithId(providedApolloClients, clientId)
    }
    return resolveDefaultClient(providedApolloClients, providedApolloClient)
 }
```

useQuery.ts: 
```
 const { resolveClient } = useApolloClient()
 const client = resolveClient(currentOptions.value.clientId)
```
 

**To Reproduce**
Use `apollo-composable`  provide Multiple ApolloClients.

**Expected behavior**
work

**Versions**
@vue/composition-api：1.0.0-beta.6
@vue/apollo-composable：v4.0.0-alpha.10
@apollo-boost：0.4.7

**Additional context**

### PR DESCRIPTION
**Pull request by liaoliaojun**
this fixes #1023

### CODE REVIEW

**Comment by shenie** [comment]
@Akryum would you be able to have a look at this PR pls?

We use multiple apollo clients and the bug is preventing us from upgrading to composition-api@1.x.

Thanks

File: packages/vue-apollo-composable/src/useApolloClient.ts [hunk 1]
```diff
@@ -37,9 +37,12 @@ export function useApolloClient<TCacheShape = any>(clientId?: ClientId): UseApol
   const providedApolloClients: ClientDict<TCacheShape> = inject(ApolloClients, null)
   const providedApolloClient: ApolloClient<TCacheShape> = inject(DefaultApolloClient, null)
 
-  function resolveClient() {
+  function resolveClient(resolvedClientId?: ClientId) {
+    if (resolvedClientId) {
+      clientId = resolvedClientId
```
Comment (paulpdaniels) [inline packages/vue-apollo-composable/src/useApolloClient.ts:42]: Won't this reassign this value for *all* subsequent requests? I think there needs to be a local variable here:

```javascript
const id = resolvedClientId ? resolvedClientId : clientId
```

Comment (paulpdaniels) [inline packages/vue-apollo-composable/src/useApolloClient.ts:42]: Doesn't this need a local variable to avoid overriding this for all subsequent requests?

Comment (liaoliaojun) [reply packages/vue-apollo-composable/src/useApolloClient.ts:42]: Yes, you are right

**Comment by Akryum** [comment]
A return was missing.

**Pull request closed without merging**